examples/dotnet/Program.cs

```diff
@@ -0,0 +1,55 @@
+using Lightrun;
+
+namespace HelloWorld
+{
+
+    class Program
+    {
+        static void Main(string[] args)
+        {
+
+            var watch = new System.Diagnostics.Stopwatch();
+
+            LightrunAgent.Start(new AgentOptions {
+                Secret = Environment.GetEnvironmentVariable("LIGHTRUN_KEY"),
+                ServerUrl= new Uri("https://app.lightrun.com/"),
+                // AlsoLogToStandardError=true,
+                // VerbosityLevel = 1,
+            });
+
+
+            System.Threading.Thread.Sleep(15000);
+
+            Console.WriteLine("Start Test");
+
+            watch.Start();
+
+            for (int i = 10; i < 1000; i++) {
+                // Console.WriteLine("Checking i " + i);
+                System.Threading.Thread.Sleep(5);
```

Comment:
I may have been involved in introducing this error, but per lines 21 and 35, sleep takes milliseconds, so this should probably be 500 (1/2 second), sleeping 5 millis isn't meaningful.

Reply:
And same for line 41.